heron/dsl/src/java/com/twitter/heron/dsl/SerializableBiFunction.java

```diff
@@ -0,0 +1,29 @@
+//  Copyright 2017 Twitter. All rights reserved.
```

Comment:
BiFunction - should be Bifunction ?

Reply:
I'm just following java.util.function terminology as defined in https://docs.oracle.com/javase/8/docs/api/java/util/function/BiFunction.html